src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -41,17 +107,23 @@ public void keyServerSetup(String keyServerIp, String keyServerHttpPort) {
     this.keyServerHttpPort = keyServerHttpPort;
   }
 
+  /**
+   * Setup the server with use.sh
+   */
   private void keyServerSetup() {
-    // Setup the server with use.sh
-    executeP2pScript("use.sh", keyServerIp, keyServerHttpPort);
+    keyServerSetup(keyServerIp, keyServerHttpPort);
+
+    String command = pathToResourceDocker + "/use.sh " + keyServerIp + " " + keyServerHttpPort;
```

Comment:
I build the String like this, as your SystemCommander takes only strings with all in there.
So I created a helper method to create those string, so I don' have to handle the output-value multiple time. 
You can change this whenever you like.